storage/localstore/mode_set.go

```diff
@@ -174,6 +174,44 @@ func (db *DB) set(mode chunk.ModeSet, addr chunk.Address) (err error) {
 			gcSizeChange = -1
 		}
 
+	case chunk.ModePinChunk:
+
+		// Get the existing pin counter of the chunk
+		existingPinCounter, err := db.GetPinCounterOfChunk(item.Address)
+		if err != nil {
```

Comment:
unnecessary and also incorrect to ignore other error than not found

Reply:
I am not ignoring the error.. This is how i find if it needs to be added as a new counter entry or to increment the existing counter entry.  Added comment to make this clear.

Reply:
Yeah.. good point. Done.

Reply:
you are ignoring the error. Whatever it is you set existingPinCounter  to 0 which is the zero value anyway and therefore unnecessary.

Reply:
Fixed this in the latest Janos review fix to initialise for levelDB.ErrNotFound and returning all other errors.